lib/testmap.py

```diff
@@ -257,9 +257,13 @@ def contexts(image, *scenarios: Iterable[str], repo: Optional[str] = None):
     "fedora-rawhide": [
         "fedora-rawhide-boot@rhinstaller/anaconda-webui"
     ],
+    "fedora-eln": [
+        "fedora-eln-boot@rhinstaller/anaconda-webui"
+    ],
     # Anaconda payload updates can affect tests
     "fedora-rawhide-anaconda-payload": [
         "fedora-rawhide-boot@rhinstaller/anaconda-webui"
+        "fedora-eln-boot@rhinstaller/anaconda-webui"
```

Comment:
Hmm something went wrong with a rebase, this does not belong here.

Reply:
should be fixed now!